app/incident/edit/controller.js

```diff
@@ -205,7 +205,7 @@ export default AbstractEditController.extend(IncidentStatuses, FriendlyId, Patie
 
     showDeleteAttachment(attachment) {
       let i18n = get(this, 'i18n');
-      let message = i18n.t('incident.messages.deleteAttachment');
+      let message = i18n.t('messages.delete_singular', { name: 'attachment' });
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.